vault/resource_database_secret_backend_connection_test.go

```diff
@@ -1194,8 +1217,22 @@ resource "vault_database_secret_backend_connection" "test" {
   redshift {
 	  connection_url = "%s"
   }
-}
-`, path, name, connURL)
+}`, name, connURL)
+	} else {
+		ret += fmt.Sprintf(`
+resource "vault_database_secret_backend_connection" "test" {
+  backend = "${vault_mount.db.path}"
```

Comment:
Should be in HCL2, no special variable interpolation required.

Reply:
Fixed in 57a0ec7